### ISSUE TITLE
Error launching SheepIt Java client on headless Ubuntu 20.04 LTS Focal Fossa

### ISSUE THREAD

**Issue by pcgeek86** [laurent-clouet/sheepit-client#261]
I'm trying to launch this command on a cloud VM, and I'm getting the following error:

```
ubuntu@ip-10-5-1-124:~$ sudo java -jar sheepit-client-6.1747.0.jar -ui text -login asdfasdfasdfasdfasdf -password asdfasdfasdfasdfadsaf -cache-dir /tmp/sheepit
Jun 15 17:52:39 Starting
Jun 15 17:52:41 Requesting Job
Jun 15 17:52:42 Frames remaining: 23350
Jun 15 17:52:42 Credits earned: 0
Jun 15 17:52:42 Rendering project "Can Blender be launched?"
Jun 15 17:52:42 Reusing cached renderer
Jun 15 17:52:43 Downloading project 100% [====================>] 0MB
Jun 15 17:52:43 Extracting renderer
Jun 15 17:52:47 Extracting project
Jun 15 17:52:47 Rendering
Jun 15 17:52:47 Compute method: CPU
Jun 15 17:52:47 Rendering (remaining )
Jun 15 17:52:47 Rendering
Jun 15 17:52:47 Compute method:
ERROR: Jun 15 17:52:47 Failed to launch renderer. Please check if you have necessary libraries installed and if you have enough free space in your working directory.
```

Any ideas how to resolve this? What libraries am I missing? I have 50 GB of free space, so plenty of extra beyond the 500 MB project size limit.

Sometimes I get this error too:

```
Jun 15 17:55:23 Starting
Jun 15 17:55:24 Requesting Job
Jun 15 17:55:24 Frames remaining: 22990
Jun 15 17:55:24 Credits earned: 0
Jun 15 17:55:24 Rendering project "Can Blender be launched?"
Jun 15 17:55:42 Downloading renderer 100% [====================>] 214MB
Jun 15 17:55:44 Downloading project 100% [====================>] 0MB
Jun 15 17:55:44 Extracting renderer
Jun 15 17:55:49 Extracting project
Jun 15 17:55:49 Rendering
Jun 15 17:55:49 Compute method: CPU
Jun 15 17:55:49 Rendering (remaining )
Jun 15 17:55:49 Rendering
Jun 15 17:55:49 Compute method:
ERROR: Jun 15 17:55:49 Renderer has generated no output file, possibly a wrong project configuration or you are missing required libraries. Will try another project in a few minutes.
```

**Comment by MCOfficer** [comment]
Without going into the specifics, installing the blender package should give you all the dependencies you need.

**Comment by pcgeek86** [comment]
I had installed just the OpenJDK 14 "headless" package. I removed the "headless" install of OpenJDK 14 and installed the regular one instead. Coincidentally, it seems to have installed the "headless" package again, along with some additional packages that resolved the dependency problem.

```
sudo apt-get install openjdk-14-jdk --yes
```

The SheepIt Java client automatically downloads and installs Blender on the system, so that isn't a required manual step.

FYI this virtual machine is running on Amazon EC2 (AWS).
